test/e2e/framework/util.go

```diff
@@ -585,7 +585,9 @@ func StartCmdAndStreamOutput(cmd *exec.Cmd) (stdout, stderr io.ReadCloser, err e
 	if err != nil {
 		return
 	}
-	Logf("Asynchronously running '%s %s'", cmd.Path, strings.Join(cmd.Args, " "))
+	// cmd.Args contains command itself as 0th argument, so it's sufficient to
```

Comment:
Devil's advocate: it is valid to invoke binary `foo` with `bar` as its `argv[0]`.

But I suppose we don't do that, so this seems look okay to me.

/lgtm
/hold

In case someone feels differently about this. Feel free to lift at your discretion.